tictactoe/player.py

```diff
@@ -25,21 +32,24 @@ def __init__(self):
         super().__init__(name=random_name)
 
     def get_turn(self, board):
-        available_cells = []
+        available_turns = []
         for i, row in enumerate(board):
             for j, column in enumerate(row):
                 if board[i][j] == Cell.EMPTY:
-                    cell_index = i * len(board) + j
-                    available_cells.append(cell_index)
-        return choice(available_cells)
+                    t = Turn(i, j)
+                    available_turns.append(t)
+        return choice(available_turns)
 
 
 class ConsolePlayer(Player):
     def __init__(self, name="Console Player"):
         frontend = ConsoleFrontend()
         super().__init__(name=name, frontend=frontend)
 
-    def get_turn(self, board) -> int:
+    def get_turn(self, board) -> Turn:
```

Comment:
Here's another type hint that snuck through 🙂

```suggestion
    def get_turn(self, board):
```